possel/data/frontend/react/login.jsx

```diff
@@ -0,0 +1,36 @@
+var LoginField = React.createClass({
+    displayName: 'LoginField',
+
+    changeHandler: function(e) {
+        var state = {};
+        state[e.target.name] = e.target.value;
+        this.setState(state)
+    },
+    keyHandler: function(e) {
+        if (e.which === 13) {
+            possel.events.flux.dispatch({
+                actionType: possel.events.action.AUTH,
+                data: {
+                    user: this.state.user,
+                    pass: this.state.pass
+                }
+            })
+        }
+    },
+    getInitialState: function() {
+        return {
+            user: 'moredhel',
```

Comment:
I'd prefer to have a controlled version of bootstrap so it works offline;
Need to do the same with FontAwesome, JQuery etc

On Fri, 27 Nov 2015 at 16:56 Kit Barnes <notifications@github.com> wrote:

> In possel/data/frontend/react/login.jsx
> <https://github.com/possel/possel/pull/22#discussion_r46061572>:
>
> > +        this.setState(state)
> > +    },
> > +    keyHandler: function(e) {
> > +        if (e.which === 13) {
> > +            possel.events.flux.dispatch({
> > +                actionType: possel.events.action.AUTH,
> > +                data: {
> > +                    user: this.state.user,
> > +                    pass: this.state.pass
> > +                }
> > +            })
> > +        }
> > +    },
> > +    getInitialState: function() {
> > +        return {
> > +            user: 'moredhel',
>
> probably want to just leave this blank !resolve
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/possel/possel/pull/22/files#r46061572>.
>